src/components/MishnaView/ManuscriptPopup.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { iManuscriptPopup } from '../../types/types';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Modal from '@mui/material/Modal';
+import { setManuscriptPopup } from '../../store/actions/mishnaViewActions';
+
+const sx = {
+  root: {
+    position: 'absolute' as 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: 400,
+    bgcolor: 'background.paper',
+    border: '2px solid #000',
+    boxShadow: 24,
+    p: 4,
+  },
+};
+
+const mapStateToProps = (state) => ({
+  manuscriptPopup: state.mishnaView.manuscriptPopup,
+});
+
+const mapDispatchToProps = (dispatch) => ({
+  setManuscriptPopup: () => {
+    dispatch(setManuscriptPopup(null));
+  },
+});
+
+export interface iProps {
+  manuscriptPopup: iManuscriptPopup;
+  setManuscriptPopup: () => void;
+}
+
+const ManuscriptPopup = (props: iProps) => {
+  const { manuscriptPopup, setManuscriptPopup } = props;
+  return (
+    <Modal
```

Comment:
Let's now add the ZoomImage component inside the popup.
You can try to merge to TAL-93/zoom-image branch into your branch.
First just set a random image into the ZoomImage. Later we will connect it to real data.